backend/tracim_backend/views/core_api/account_controller.py

```diff
@@ -96,6 +96,7 @@ def set_account_infos(self, context, request: TracimRequest, hapic_data=None):
             lang=hapic_data.body.lang,
             do_save=True
         )
+        uapi.execute_updated_user_actions(user)
```

Comment:
same remark: this must be implicit. Do not call yourself the on_user_updated() method (by the way, rename `execute_updated_user_actions` to `on_user_updated`

Reply:
se: https://github.com/tracim/tracim/pull/1578#discussion_r274287675

Reply:
https://github.com/tracim/tracim/issues/1589